### ISSUE TITLE
Royal Tunnel

### ISSUE THREAD

**Issue by Kazuumaruu** [warpKaiba/TMpokeheroes#5]
I've been getting the "Timed Out" thingie for the Royal Tunnel lately. Is it possible that with the release of the new Galar pokemon, that it's messing up the script, and if so, would you mind looking into it for us? ;w;

**Comment by MopeyBoop** [comment]
I have looked at it a bit more closely - the only thing it seems to have trouble with is the question for which level a Pokémon levels up at, but not Dex Entry, Egg group or Type. It is only the evolution level which seems to be a big hiccough for the script.

**Comment by MopeyBoop** [comment]
I will attempt to see how much I can add to line 211 of the script - I will assemble the information, and maybe, if I have the time and brainpower, also double check the existing info in line 211 for possible errors, as sometimes, it still throws up the "wrong answer".

**Comment by Kazuumaruu** [comment]
Awesome, thank you so much ;w;

**Comment by MopeyBoop** [comment]
I will be slow, to be fair, but I will try my hardest to make progress in the next couple days. Assembling the information will be the most work, but it will be worthwhile.

**Comment by MopeyBoop** [comment]
"810	Grookey	Grass	Chimp	Starter	?	5355	?","813	Scorbunny	Fire	Rabbit	Starter	?	5355	16","816	Sobble	Water	Water Lizard	Starter	?	5355	16","821	Rookidee	Flying	Tiny Bird	Easy	Flying	4080	18","822	Corvisquire	Flying	Raven	Easy	Flying	4080	34","823	Corviknight	Flying/Steel	Raven	Easy	Flying	4080	-","835	Yamper	Electric	Dog	Medium	Field	5355	25","836	Boltund	Electric	Dog	Medium	Field	5355	-"


Okay, this is what I got up to this point. If you add this to the end of Line 211, while making sure to preserve the ]; it should work. I'm currently testing it and will add more data if needed, especially since another small batch dropped on PH.

**Comment by MopeyBoop** [comment]
"822	Corvisquire	Flying	Raven	Easy	Flying	4080	38"

My bad, Corvisquire apparently is not 34, but 38. Info is still a smidge unreliable with Gen 8.

**Comment by Kazuumaruu** [comment]
For some reason, it's still having trouble with Corvisquire xD I have it scripted as 38, but I had to still manually click on the 38 to get it to progress

**Comment by MopeyBoop** [comment]
Huh, weird. Did you double check that every set of data is enclosed like ">Corvisquire info<" and is separated by a comma from the other data sets?

**Comment by Kazuumaruu** [comment]
I copied what you had and just changed the 34 to 38. So yes ;w; Everything else seems to work, it just has a hard time understanding the level. That's it.

**Comment by Kazuumaruu** [comment]
I just had the same problem with Yamper as well. So it just seems to be the leveling that messes it up on my end ;3;

**Comment by MopeyBoop** [comment]
Hm. Can you copy me what your line 211 looks like? Maybe I can figure it out - I'm by far not a coder, but maybe I can spot some kind of mistake that would explain the difficulties. It's enough to just show me the end batch with all the Galar data.

**Comment by Kazuumaruu** [comment]
"810 Grookey Grass Chimp Starter ? 5355 ?","813    Scorbunny Fire Rabbit Starter ? 5355 16","816 Sobble Water Water Lizard Starter ? 5355 16","821    Rookidee Flying Tiny Bird Easy Flying 4080 18","822 Corvisquire Flying Raven Easy Flying 4080 38","823 Corviknight Flying/Steel Raven Easy Flying 4080 -","831 Wooloo Normal Sheep Medium Field 4080 24","831 Dubwool Normal Sheep Medium Field 4080 -","833 Chewtle Water Snapping Medium Monster/Water 1 5355 22","834 Drernaw Water/Rock Bite Medium Monster/Water 1 5355 -","835 Yamper Electric Puppy Medium Field 5355 25","836 Boltund Electric Dog Medium Field 5355 -","870 Falinks Fighting Formation Hard Fairy/Mineral 6630 -"

I'm not a coder at all either, so don't laugh too much xD I just tried to copy what I thought each of the things were from their Dex Entries on PH. I think i copied yours and just added the newer mons but thats what I got ;w;

**Comment by MopeyBoop** [comment]
Did you also add the ]; at the end of the line? Or maybe you've accidentally doubled it? I'm a bit stumped myself.

**Comment by Kazuumaruu** [comment]
I dont think so ;3; I reinstalled the file from here, and just went in and copy pasted all that at the end like you said.  So I have no clue what I'm doing wrong

**Comment by MopeyBoop** [comment]
Huh. Okay, that is double weird. Uuuuh, I'll try something. If this upload works, it should give you a working script - I broke down Line 211 so that all data is more easily legible. (I hope this works, there is not much indication before posting that it is working.) You can copy paste the complete thing from the file into tampermonkey and then save it. 
[Royal Tunnel Script Line 211 Broken Down.txt](https://github.com/warpKaiba/TMpokeheroes/files/4007419/Royal.Tunnel.Script.Line.211.Broken.Down.txt)

**Comment by Kazuumaruu** [comment]
Testing it out now ;w; Thank you again so much for your help! I feel like maybe I made that a bit harder for you then it should have been, so I apologize ;3; but I only have one more question. With this script, do I just add a new line for each pokemon as they're released on PH? I don't have to do anything special other than getting the information from the Dex Entry?

**Comment by MopeyBoop** [comment]
Yup, you can just add new lines, just make sure to keep the format consistent. Should work just fine. :)

**Comment by DivergentV** [comment]
hey there are new questions in royal tunnel can you help by updating adding their weight and height in the script .
